src/components/contacts/contactsPage.jsx

```diff
@@ -6,30 +6,34 @@ var CreateContact 	= require('./add-contact.jsx');
 var ViewContact 	= require("./view-contact.jsx");
 var Header 			= require("../header/header.jsx");
 var SearchBox 		= require("../orders/search-box.jsx");
+var ViewReminders	= require("../reminders/view-reminder.jsx");
+
+var getWeek			= require("../../lib/getWeek.js");
```

Comment:
no difference the server/lib.groupBy.js should have been deleted thanks for noticing